libs/blocks/aside/aside.css

```diff
@@ -429,15 +642,29 @@
     bottom: 0;
   }
 
-  .aside.split-right .split-image img {
+  .aside.split .tablet-wide img,
+  .aside.split .tablet-wide video {
+    aspect-ratio: var(--aspect-ratio-wide);
+  }
+  
+  .aside.split .tablet-standard img,
+  .aside.split .tablet-standard video {
+    aspect-ratio: var(--aspect-ratio-standard);
+  }
```

Comment:
I see a `square` [option for mobile](https://github.com/adobecom/milo/pull/979/files#diff-172ebed29740bfd3c497f52bd6e25f9eb65802501dafd514cb208bba9f12d0d0R178). No square option for tablet?

Reply:
@rgclayton As per the design specs shared by the consonant team the square option is available for mobile view only hence we have not added the option for tablet or desktop.